### PR TITLE
prevent concatenation of rake lint params to npm lint params

### PR DESCRIPTION
**Pull request by mpugach**
[Review on Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/299)

### CODE REVIEW

File: lib/generators/react_on_rails/templates/base/base/client/server.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ server.app.use('/', (req, res) => {
 server.listen(PORT, 'localhost', err => {
   if (err) console.error(err);
   console.log(
-    '=> 🔥  Webpack development server is running on port ' + PORT
+    `=> 🔥  Webpack development server is running on port ${PORT}`
```
Comment (mpugach) [inline lib/generators/react_on_rails/templates/base/base/client/server.js:67]: now this is spotted by linters

File: lib/generators/react_on_rails/templates/base/base/lib/tasks/linters.rake.tt [hunk 2]
```diff
@@ -57,14 +57,14 @@ if %w(development test).include? Rails.env
...
-      cmd = "cd client && npm run eslint . -- --ext .jsx,.js"
+      cmd = "cd client && npm run eslint"
       puts "Running eslint via `#{cmd}`"
       sh cmd
     end
 
     desc "jscs"
     task :jscs do
-      cmd = "cd client && npm run jscs ."
+      cmd = "cd client && npm run jscs"
```
Comment (mpugach) [inline lib/generators/react_on_rails/templates/base/base/lib/tasks/linters.rake.tt:67]: prevents double pass

File: lib/generators/react_on_rails/templates/base/base/lib/tasks/linters.rake.tt [hunk 3]
```diff
@@ -57,14 +57,14 @@ if %w(development test).include? Rails.env
     <%- unless options.skip_js_linters? -%>
     desc "eslint"
     task :eslint do
-      cmd = "cd client && npm run eslint . -- --ext .jsx,.js"
```
Comment (mpugach) [inline lib/generators/react_on_rails/templates/base/base/lib/tasks/linters.rake.tt]: these params are already in `client/package.json`

File: lib/generators/react_on_rails/templates/base/base/lib/tasks/linters.rake.tt [hunk 4]
```diff
@@ -57,14 +57,14 @@ if %w(development test).include? Rails.env
...
+      cmd = "cd client && npm run eslint"
```
Comment (mpugach) [inline lib/generators/react_on_rails/templates/base/base/lib/tasks/linters.rake.tt:60]: these params are already in `client/package.json`

**Comment by justin808** [comment]
:lgtm: Once we pass CI, I'll merge. **BIG** Thanks @mpugach!

---

Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

*[lib/generators/react_on_rails/templates/base/base/client/server.js, line 67 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/299#-KBUXwnbITHhbpVRLt78-r1-67:-KBV0p5vdb_JVENyEggY:262468467) ([raw file](https://github.com/shakacode/react_on_rails/blob/2c0e90c9b55b1e4fe6356d5161f14fba9bfd6cd3/lib/generators/react_on_rails/templates/base/base/client/server.js#L67)):*
:thumbsup: 

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/react_on_rails/299#-:-KBV0yactbaHdcrMuD1c:-1839259288)*

**Comment by justin808** [comment]
Merged. Thanks @mpugach!

**Pull request merged**